docs/learn-faq.md

```diff
@@ -6,80 +6,327 @@ sidebar_label: Frequently Asked Questions (FAQs)
+protocols for finding peers and connecting to them. It's modules have logic for
+content routing, peer routing, peer discovery, different transports, and NAT
+traversals. It is intended to be used by applications for building large scale
+peer-to-peer networks by only selecting the parts of the protocol suite that are
+needed.
+
+The Rust implementation of the specification was built and primarily maintained
+by a team of contributors at Parity Technologies. The Go and JavaScript versions
+are maintained by Protocol Labs as well as community contributors. It is an open
+source project that is actively developed and expanded on various code repositories
+hosted on [GitHub][libp2p github].
+
+### Does Polkadot use libp2p?
+
+Yes, since Polkadot is built with Substrate. Substrate uses a networking protocol
+that is based on libp2p (specifically the Rust libp2p library). However, Substrate
+uses a mix of standard libp2p protocols and protocols that are homegrown and not
+official libp2p standards. Of the standards protocols, these which are shared
+with other implementations of libp2p such as IPFS, are connection-checking (ping),
+asking for information on a peer (identity), and Kademlia random walks (kad).
+
+Of the protocols that are custom to Substrate, there are the legacy Substrate
+stream, a request-response for getting information on blocks (sync), a light
+client protocol, a notification protocol for transactions, and block announcement.
+For detailed information on how Substrate uses libp2p and the standard and custom
+protocols, please see the [networking documentation][substrate network]. 
+
+### How does libp2p differ from IPFS?
+
+The [Interplanetary Filesystem][ipfs] (IPFS) is a peer-to-peer hypermedia protocol
+used primarily for storage of files. It allows one to upload a file onto the
+network and share it with its content addressable URI. IPFS, like Substrate,
+is an application of libp2p and exists higher on the technology stack. Although
+both IPFS and Substrate use libp2p, it cannot be said that Substrate "uses" IPFS
+since besides sharing the underlying library for networking there is no native
+integration between the two applications.
+
+## Kusama
+
+### What is the minimum amount of KSM I can have in my account?
+
+It is recommended to always ensure that you keep at least 0.1 KSM in your account in order to avoid the reaping threshold of 0.01 KSM.  If you have less than 0.01 KSM in your account, that account will be "reaped" - it will be removed and no longer occupy space on the chain. In other words, no accounts are allowed on-chain with an account balance of less than 0.01 KSM.   This is a dust prevention measure, in order to ensure that the chain is not full of accounts with minuscule amounts of KSM taking up space. Since the blockchain is copied to every person running a full node, any savings of space provides dramatic benefits in terms of scalability.
```

Comment:
```suggestion
We recommend users always keep at least 0.1 KSM in their account in order to avoid the reaping threshold of 0.01 KSM.  If you have less than 0.01 KSM in your account, that account will be "reaped" - it will be removed and no longer occupy space on the chain. In other words, no accounts are allowed on-chain with an account balance of less than 0.01 KSM.   This is a dust prevention measure, in order to ensure that the chain is not full of accounts with minuscule amounts of KSM taking up space. Since the blockchain is copied to every person running a full node, any savings of space provide dramatic benefits in terms of scalability.
```